tests/python/test_matrix.py

```diff
@@ -1181,3 +1181,16 @@ def foo() -> ti.i32:
         return sig.sum() + p.sum()
 
     assert foo() == 4
+
+
+@test_utils.test(debug=True)
```

Comment:
OOC: why do we need debug=True for this test btw?

Reply:
For in-kernel assertions :-)